RadarSDK/RadarEvent.m

```diff
@@ -291,11 +291,11 @@ - (instancetype _Nullable)initWithObject:(id)object {
                                              verticalAccuracy:-1
                                                     timestamp:(createdAt ? createdAt : [NSDate date])];
         }
+    }
 
-        id metadataObj = dict[@"metadata"];
```

Comment:
was this happening in the `locationObj` block before, hence the extra `}`? my fault if so 🤦🏽‍♂️

Reply:
Yeah, it was. It took me a while to figure out why the breakpoint I set on it wasn't being hit. :)

Reply:
theoretically shouldn't `metadata` still have been serialized? since the check was happening anyways, just nested within another conditional?

the fix is necessary and important, but I don't think things were "broken" beforehand?